blazor/document-editor/getting-started/server-side-application.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Getting Stared with Blazor DocumentEditor Component | Syncfusion
+title: Getting Started with Blazor DocumentEditor in WASM App | Syncfusion
```

Comment:
Use proper title for this page. this page describes server app, check description and update title as well.

Reply:
resolved